### PR TITLE
User can see instructions for trip

### PR DESCRIPTION
**Pull request by desolagiwa**
The user can click "Trip Details" to see a brief description of the trip, and then choose to start a trip. The user can also view step-by-step directions for each segment of the trip

### CODE REVIEW

**Comment by desolagiwa** [comment]
Screenshot 2024-07-16 at 5 42 25 AM
Screenshot 2024-07-16 at 5 44 25 AM

**Comment by rguan1** [comment]
Did you ever find the best practice to escape the lat and long? At min, you probably want to annotate the attributes: `startLat` & `endLong`

File: client/capstone-app/package.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
     "react": "^18.3.1",
     "react-dom": "^18.3.1",
     "react-leaflet": "^4.2.1",
-    "react-router-dom": "^6.24.0",
+    "react-router-dom": "^6.24.1",
```
Comment (rguan1) [inline client/capstone-app/package.json:22]: Why are these versions changing?

File: client/capstone-app/src/components/MapWithRoute.jsx [hunk 2]
```diff
@@ -2,21 +2,40 @@ import React from 'react';
...
+const MapWithRoute = ({ routeData, directions, centerCoordinates}) => {
   const onEachFeature = (feature, layer) => {
     if (feature.properties && feature.properties.summary) {
       layer.bindPopup(`Distance: ${feature.properties.summary.distance} meters<br>Duration: ${feature.properties.summary.duration} seconds`);
     }
   };
 
   return (
-    <MapContainer center={centerCoordinates} zoom={15} style={{ height: '80vh', width: '100vh' }}>
+    <MapContainer center={[-90.071533, 29.951065]} zoom={15} style={{ height: '60vh', width: '100vh' }}>
```
Comment (rguan1) [inline client/capstone-app/src/components/MapWithRoute.jsx:19]: Is there a reason why you reverted to hardcoded coords?

File: client/capstone-app/src/components/RouteInfo.jsx [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+           )
+           : (<ModalContent>
+              {(onClose) => (
+                <>
+                  <ModalHeader className="flex flex-col gap-1">
+                    View Trip
+                  </ModalHeader>
+                  <ModalBody>
+                  <h2>Walk {directions[0].features[0].properties.summary.distance} miles to {data.startStopName}</h2>
+                  <h2>Ride {directions[1].features[0].properties.summary.distance} miles to {data.transfers[0].endStopName}</h2>
```
Comment (rguan1) [inline client/capstone-app/src/components/RouteInfo.jsx:52]: Why is there a hardcoded walk, ride, walk, etc. here? Wouldn't it make more sense to have these components generated dynamically?

**Comment by slinakm** [comment]
Thanks for the summary of updates!

**Comment by slinakm** [comment]
Next time could you make the commits and Pull requests smaller? It would be easier to review + a great way to keep track of your progress

File: client/capstone-app/package.json (hunk 1 above)
Comment (desolagiwa) [reply client/capstone-app/package.json:22]: This was the latest version, and I wanted to keep the project updated

File: client/capstone-app/src/components/MapWithRoute.jsx (hunk 2 above)
Comment (desolagiwa) [reply client/capstone-app/src/components/MapWithRoute.jsx:19]: It was for testing purposes; it has been changed back in future PRs

File: client/capstone-app/src/components/RouteInfo.jsx (hunk 3 above)
Comment (desolagiwa) [reply client/capstone-app/src/components/RouteInfo.jsx:52]: Yes! This has been changed in [PR #6](https://github.com/desolagiwa/metaU-capstone/pull/6)

**Pull request merged**